### PR TITLE
Move qtpy import outside beamcentrefinder

### PR DESCRIPTION
**Pull request by TTitcombe**
**Description of work.**
Moves qtpy import and plotting functionality outside the SANSBeamCentreFinder algorithm.

**To test:**
1. Open workbench
2. Interfaces -> SANS -> ISIS SANS
3. Load the attached user file and batch file (ISIS data archive required)
4. In the beam centre tab and click `run`
5. After a few seconds a plot should appear. This will change as the next iteration of the algorithm completes
6. Close workbench and do the same in mantidplot. The plots will look different, but everything should work the same in general

*There is no associated issue.*

*This does not require release notes* because **internal code change**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSBeamCentreFinder.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 from mantid.api import (DataProcessorAlgorithm, MatrixWorkspaceProperty, AlgorithmFactory, PropertyMode, Progress)
 from mantid.kernel import (Direction, PropertyManagerProperty, StringListValidator, Logger)
 from mantid.simpleapi import CloneWorkspace, GroupWorkspaces
+from sans.algorithm_detail.beamcentrefinder_plotting import can_plot_beamcentrefinder, plot_workspace_quartiles
```
Comment (AndreiSavici) [inline Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSBeamCentreFinder.py]: I don't think that something in Framework should depend on something in scripts. You should instead have the interface in scripts call this algorithm and the plotting separately

**Comment by TTitcombe** [comment]
@AndreiSavici ah, SANS does this a lot in its python algorithms. I may have a not insignificant refactor on my hands :thinking:

File: Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSBeamCentreFinder.py (hunk 1 above)
Comment (martyngigg) [reply Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANS/SANSBeamCentreFinder.py]: @AndreiSavici A few algorithms do this as a way to consolidate code. It seems okay to me? Our issue really is that scripts is a bit of a mess of different bits of code.

**Review by mdoucet — APPROVED** [review]
I tested the latest version of this PR and it fixes my original issue. I can now import mantid in another application without getting a pyqt conflict.

**Pull request merged**